src/state/slices/portfolioSlice/utils.test.ts

```diff
@@ -1,13 +1,203 @@
+import {
+  assetIds,
+  btcAddresses,
+  btcCaip2,
+  btcCaip19,
+  ethCaip10s,
+  ethCaip19,
+  ethPubKeys,
+  foxCaip19,
+  mockBtcAccount,
+  mockBtcAddress,
+  mockEthAccount,
+  mockEthToken,
+  unknown1Caip19,
+  unknown2Caip19,
+  unknown3Caip19,
+  usdcCaip19,
+  yvusdcCaip19,
+  zeroCaip19
+} from 'test/mocks/accounts'
+import { bn } from 'lib/bignumber/bignumber'
+
+import { Portfolio } from './portfolioSlice'
 import {
   accountIdToChainId,
   accountIdToLabel,
   accountIdToSpecifier,
+  accountToPortfolio,
   assetIdtoChainId,
   btcChainId,
   ethChainId,
   findAccountsByAssetId
 } from './utils'
 
+const tokenBalance = (ethAccount: any, caip19: any) => {
+  return ethAccount.chainSpecific.tokens.find((token: any) => token.caip19 === caip19).balance
+}
+
+const ethAccount1 = mockEthAccount({
```

Comment:
Instead of defining the eth and btc accounts at the top level, you can use the `mockETHandBTCAccounts` helper inside `accounts.ts` if you need to change or modify any of the values of an account, the helper function accepts an object for each account. 
```
      const { ethAccount, ethAccount2, ethAccountId, ethAccount2Id } = mockETHandBTCAccounts({
        ethAccountObj: { balance: '1000000000000000000' },
        ethAccount2Obj: { balance: '200000000000000000' }
      })
```

Reply:
Done ! see my updated tests in `portfolioSlice.test.ts`. I moved the tests there as it made more sense (did not understand how snapshot worked at first look).